libs/klv/src/main/java/org/codice/ddf/libs/klv/data/numerical/KlvNumericalDataElement.java

```diff
@@ -44,6 +44,8 @@ public KlvNumericalDataElement(final byte[] key, final String name) {
    * @param errorIndicatorValue value that indicates an encoded error
    * @throws IllegalArgumentException if any arguments are null
    */
+  @SuppressWarnings(
+      "squid:S2789" /* Optional errorIndicatorValue can be null, check for null is ok */)
```

Comment:
❗️ By design, Optional should never be null. This is bad practice as they are used to avoid dealing with null

Reply:
When I looked into it all of the classes calling that constructor pass in an optional and never null, so I decided to take the useless check out.